www/authmemcookie.php

```diff
@@ -91,12 +91,25 @@
 	/* Store the authentication data in the memcache server. */
 
 	$data = '';
+	$amcConfig = SimpleSAML_Configuration::getConfig('authmemcookie.php');
+	$format = strtolower($amcConfig->getBoolean('format', 'implode'));
```

Comment:
This should be using `getString()`, as the option is a `string`, not a `boolean`.